votor/src/root_utils.rs

```diff
@@ -32,6 +35,15 @@ pub(crate) struct RootContext {
     pub(crate) drop_bank_sender: Sender<Vec<BankWithScheduler>>,
 }
 
+#[derive(Debug, Error)]
+pub(crate) enum SetRootError {
+    #[error("Failed to record optimistic slot in blockstore: {0}")]
+    RecordSlotError(#[from] BlockstoreError),
+
+    #[error("Error sending bank nofification: {0}")]
+    SendNotificationError(#[from] SendError<()>),
```

Comment:
I was skimming through the codebase a bit. I think we generally try to avoid using `Error` suffix in error enum variants as it is arguably redundant. Ie, `SetRootError:: SendNotificationError` includes `Error` twice.

Looking at `ErrorKind` here, I see they avoid it too:
https://doc.rust-lang.org/std/io/enum.ErrorKind.html

We definitely have some errors in the codebase that do not adhere to this, but I generally like avoiding the double up. Given that we're not consistent, I'm fine if you want to leave as is too.